### PR TITLE
Update PolyMC install instructions

### PR DESCRIPTION
**Pull request by ZekeSmith**
Simple changes to update instructions for modern versions of PolyMC.

Outline changes:
* Reword title to better reflect PolyMC's goals and current implementation
* Update install instructions to use new instance mod loader selection
*  Update installing mods to better reflect installation process
*  Add Updating mods section

### CODE REVIEW

File: src/install/polymc.md [hunk 1]
```diff
@@ -9,20 +9,16 @@ permalink: /install/polymc/
...
 3. Fill out the **Name** and **Group** boxes as required
-4. Select **Vanilla** from the list on the left, and pick the version of Minecraft you'd like to use for your instance
-5. Click **OK** to create a new vanilla instance
-6. Right-click on the instance and select **Edit Instance**, or click on it once and select **Edit Instance** from the 
-   menu on the right of the window
-7. Select **Version** from the list on the left, and click the **Install Quilt** button in the menu on the right side 
-   of the window
-8. Select the Quilt Loader version you'd like to use and click **OK**
-9. Set up the rest of your instance as normal
+4. Select **Custom** *(**Vanilla** for versions before 5.0)* from the list on the left, and pick the version of Minecraft you'd like to use for you instance
```
Comment (DioEgizio) [inline src/install/polymc.md:18]: 5.0 isn't even released yet

File: src/install/polymc.md [hunk 2]
```diff
@@ -31,16 +27,20 @@ mods and get playing.
...
-   select **Edit Instance** from the menu on the right of the window
-2. Select **Mods** from the list on the left, and click on the **Download mods** button in the menu on the right side
-   of the window
-3. Find a Fabric or Quilt mod from CurseForge or Modrinth, and click the **OK** button
-4. Ensure you have the **Quilt Standard Libraries** installed, which you can find
-   [on Modrinth](https://modrinth.com/mod/qsl)
+1. Right-click on the instance you'd like to install mods to and select **Edit Instance**, or click on it once and select **Edit Instance** from the menu on the right of the window
+2. Select **Mods** from the list on the left, and click on the **Download mods** button in the menu on the right side of the window
+3. Find a mod from CurseForge or Modrinth, and click the **Select mod for download** button
+4. Ensure you have selected **Quilt Standard Libraries** to be installed with the mod downloader, alternatively download [on Modrinth](https://modrinth.com/mod/qsl)
```
Comment (DioEgizio) [inline src/install/polymc.md:33]:
```suggestion
4. Ensure you have selected **Quilted Fabric API** to be installed with the mod downloader, alternatively download [on Modrinth](https://modrinth.com/mod/qsl)
```

Comment (ZekeSmith) [reply src/install/polymc.md:33]: Right I'll ammend this to be up to date, im pretty sure the other pages also refer to qsl so I might add that into the pr.

**Comment by gdude2002** [comment]
Sorry for the delay.

While I appreciate what you're doing here, two things to note:

1. The launcher pages should not read like advertisements. That's a very important part of this section of the site.
2. It's unlikely we'll be able to merge this PR, as the site rebuild in #58 is very close to being complete - and this will completely break the PR.

I appreciate this as a notification that the processes involved with working with PolyMC have changed, though - we do need to update this page post-rebuild.

I'm going to close the PR for now, as we will not be able to merge it. We'll revisit this page post-rebuild.

**Pull request closed without merging**